src/test/java/com/rb/nonbiz/testmatchers/RBArrayMatchers.java

```diff
@@ -26,7 +26,11 @@ protected boolean matchesSafely(T[] actual) {
         }
         return IntStream
             .range(0, expected.length)
-            .allMatch(i -> itemMatcherGenerator.apply(expected[i]).matches(actual[i]));
+            .allMatch(i -> {
+              // The following is un-inlined to make it easier to place breakpoints during debugging, when needed.
+              boolean matches = itemMatcherGenerator.apply(expected[i]).matches(actual[i]);
+              return matches;
```

Comment:
we rely on automated testing instead of debugging. But sometimes we have to debug automated tests themselves.